OloEngine/src/OloEngine/Renderer/Font.cpp

```
@@ -69,11 +77,27 @@ namespace OloEngine
             return;
         }
 
+        // m_Path stays as set above; LoadFromMemory deliberately leaves it alone
+        // so the file path survives the shared parse.
+        LoadFromMemory(filepath.filename().stem().string(), fontBuffer, ranges);
+    }
+
+    void Font::LoadFromMemory(std::string name, std::span<const u8> fontData, const std::vector<FontCodepointRange>& ranges)
+    {
```

Cppcheck (2.20.0)

[style] The function 'GetAtlasTexture' is never used.

(unusedFunction)